packages/manager/modules/pci/src/projects/project/kubernetes/details/service/reset-kubeconfig/reset-kubeconfig.html

```diff
@@ -7,7 +7,7 @@
     <oui-modal
         data-primary-label="{{ ::'pci_projects_project_kubernetes_service_reset_kubeconfig_common_confirm' | translate }}"
         data-primary-disabled="!$ctrl.isClusterReady"
-        data-secondary-action="$ctrl.goBack()"
+        data-secondary-action="$ctrl.onResetKubeModalCancel()"
```

Comment:
Cannot find `onResetKubeModalCancel()` being used.

Reply:
Right @varun257, thanks for observation.

Reply:
Fixed.